net/radsecproxy/+POST_INSTALL

```diff
@@ -0,0 +1,4 @@
+if /usr/local/etc/rc.d/configd status > /dev/null; then
```

Comment:
Do you really need this file?

Reply:
On my development-machine the menu-items won't show up without `configure_plugins`.
The `config restart` is part of the [tutorial](https://docs.opnsense.org/development/examples/helloworld.html#create-an-installable-plugin).